astarte-device-sdk/AstarteDeviceSDK.h

```diff
@@ -66,6 +66,9 @@ class AstarteDeviceSDK : public Hemera::AsyncInitObject
 
     ConnectionStatus connectionStatus() const;
 
+    bool connectToBroker();
+    bool disconnectFromBroker();
```

Comment:
I'm not 100% sure if we should mention here `Broker` or just find a more generic naming, such as `connectToAstarte()`.